oxidation/client/src/components/ProfileHeader.vue

```diff
@@ -0,0 +1,121 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-item
+    button
+    id="click-trigger"
+    class="container"
+    @click="isPopoverOpen = !isPopoverOpen; openPopover($event)"
+  >
+    <ion-avatar slot="start" class="avatar">
+      <img
+        alt="Silhouette of a person's head"
+        src="https://ionicframework.com/docs/img/demos/avatar.svg"
+      >
+    </ion-avatar>
+    <div class=" text-icon">
```

Comment:
Why the space here?

Reply:
What space are you talking about? The indent?

Reply:
`<div class=" text-icon">` There's a space before ` text-icon`.

Reply:
I removed the useless space.